@ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchRefinementListFilter/index.js

```diff
@@ -42,6 +42,7 @@ SearchRefinementListFilter.propTypes = {
 
 SearchRefinementListFilter.defaultProps = {
   operator: 'OR',
+  size: '20',
```

Comment:
@jonhartzler @jmille79 I don't know if this is a sensible default value, but we needed something set.

Reply:
set 10.